config/interface/input_elements/demand_buildings_heating.yml

```diff
@@ -63,4 +63,11 @@
   position: 9
   dependent_on: has_coal_oil_for_heating_built_environment
   slide_key: demand_buildings_heating
+- key: buildings_space_heater_heatpump_air_water_electricity_share
+  share_group: heating_buildings
+  step_value: 0.1
+  unit: "%"
+  related_node: buildings_space_heater_heatpump_air_water_electricity
+  position: 6
+  slide_key: demand_buildings_heating
```

Comment:
Little bit nitpicky, but maybe we could move the new slider up in position a bit, so that it is together with the other heat pumps?
<img width="337" alt="Screenshot 2021-08-05 at 16 07 44" src="https://user-images.githubusercontent.com/14875123/128364412-5c36c85b-2a08-433d-bcda-b76299ef20d0.png">

Reply:
Good point @noracato!